backend/github.js

```diff
@@ -96,3 +96,61 @@ exports.deleteHook = function (name, hook, accessToken, callback) {
     }
   });
 };
+
+/**
+ * Get all hooks registered to a repository
+ * @param name: `full_name` of the repository
+ * @param accessToken: Access token
+ * @param callback
+ */
+exports.getRepositoryHooks = function (name, accessToken, callback) {
+  request({
+    url: 'https://api.github.com/repos/' + name + '/hooks',
+    headers: {
+      'User-Agent': 'Yaydoc',
+      'Authorization': 'token ' + crypter.decrypt(accessToken)
+    }
+  }, callback);
+};
+
+/**
+ * Register a hook to the repository for Yaydoc
+ * @param name: `full_name` of the repository
+ * @param accessToken: Access token
+ * @param callback
+ */
+exports.registerHook = function (name, accessToken, callback) {
```

Comment:
Anything i write will be overwritten by your implementation. I don't think there is a need to update it now